nucypher/blockchain/eth/scripts/estimate_gas_tester.py

```diff
@@ -15,74 +15,62 @@
 
 
 def estimate_gas():
-    solidity_compiler = SolidityCompiler(test_contract_dir=CONTRACTS_DIR)
-
-    # create a temporary registrar for the tester blockchain
-    temporary_registry = TemporaryEthereumContractRegistry()
+    """
+    Deploy contracts in tester.
+    A simple Python script to deploy contracts and then estimate gas for different methods.
+    """
 
-    # Configure a custom provider
-    overrides = {'gas_limit': 4626271}
-    pyevm_backend = OverridablePyEVMBackend(genesis_overrides=overrides)
+    solidity_compiler = SolidityCompiler(test_contract_dir=TEST_CONTRACTS_DIR)
 
-    eth_tester = EthereumTester(backend=pyevm_backend, auto_mine_transactions=True)
-    pyevm_provider = EthereumTesterProvider(ethereum_tester=eth_tester)
+    # create a temporary registrar for the tester blockchain
+    memory_registry = InMemoryEthereumContractRegistry()
 
     # Use the the custom provider and registrar to init an interface
-    circumflex = BlockchainDeployerInterface(compiler=solidity_compiler,  # freshly recompile
-                                             registry=temporary_registry,  # use temporary registrar
-                                             providers=(pyevm_provider,))  # use custom test provider
+    deployer_interface = BlockchainDeployerInterface(compiler=solidity_compiler,  # freshly recompile if not None
+                                                     registry=memory_registry,
+                                                     provider_uri='tester://pyevm')
 
     # Create the blockchain
-    testerchain = TesterBlockchain(interface=circumflex, test_accounts=10)
+    testerchain = TesterBlockchain(interface=deployer_interface,
+                                   test_accounts=DEFAULT_NUMBER_OF_URSULAS_IN_DEVELOPMENT_NETWORK,
+                                   airdrop=False)
     origin, ursula1, ursula2, ursula3, alice1, *everyone_else = testerchain.interface.w3.eth.accounts
-    circumflex.deployer_address = origin  # Set the deployer address from a freshly created test account
+    deployer_interface.deployer_address = origin  # Set the deployer address from a freshly created test account
 
     token_deployer = NucypherTokenDeployer(blockchain=testerchain, deployer_address=origin)
 
     token_deployer.deploy()
     token_agent = token_deployer.make_agent()
 
-    miners_escrow_secret = os.urandom(constants.DISPATCHER_SECRET_LENGTH)
+    miners_escrow_secret = os.urandom(int(constants.DISPATCHER_SECRET_LENGTH))
     miner_escrow_deployer = MinerEscrowDeployer(
-        token_agent=token_agent,
         deployer_address=origin,
         secret_hash=testerchain.interface.w3.sha3(miners_escrow_secret))
 
     miner_escrow_deployer.deploy()
     miner_agent = miner_escrow_deployer.make_agent()
 
-    policy_manager_secret = os.urandom(constants.DISPATCHER_SECRET_LENGTH)
+    policy_manager_secret = os.urandom(int(constants.DISPATCHER_SECRET_LENGTH))
```

Comment:
Was this producing a type problem?  This may have been based on alder version of the codebase  / constant-sorrow, if so.  In fact, we stopped using constant sorrow for this value.

Reply:
There was no type problem (was some import problem), it was left after trying to understand where the problem is. Where is now proper constant?

Reply:
#594 fixes this, so `vodka` will be rebased over top of master after merging #594

Reply:
I've just rebased `vodka` on top of master (#594 was merged recently). I think you can rebase yours again on top of vodka now.